### DISCUSSION TITLE
Multiple IPs with the same PTR hostname

### DISCUSSION THREAD

**Discussion by Fanny12333** [postalserver/postal#2366] [Development discussions]
I tried to do IP rotation with several IPs of the same PTR Hostname, after I tried to send an email the PTR was not detected, sometimes detected sometimes not.

Maybe there is the same experience?

https://imgur.com/a/InSpTl5

**Comment by willpower232** [comment]
PTR records are maintained by whoever is in charge of the IP addresses which is normally your server provider. You should confirm with them that they have set up a PTR for each IP address they have given to you.
